docker-compose.yml

```diff
@@ -33,7 +33,8 @@ services:
     command: mkcert -cert-file "${DEV_URL:-local.devgo.vip}.crt" -key-file "${DEV_URL:-local.devgo.vip}.key" "${DEV_URL:-local.devgo.vip}" "*.${DEV_URL:-local.devgo.vip}"
 
   db:
-    image: mariadb:10.3
+    image: mysql:8
```

Comment:
Can you check if the entry point supports passing just the config flags? Most containers support that.

Reply:
It does support it, pushed the change